test/integration/glob.spec.js

```diff
@@ -28,7 +28,7 @@ describe('globbing', function() {
           expect(
             results.stderr,
             'to contain',
-            'cannot find any files matching pattern "./*-none.js"'
+            '✖ Cannot find any files matching pattern "./*-none.js"'
```

Comment:
can you pull in the symbol from `log-symbols`? this may fail on windows

Reply:
Or just leave it and the pattern out...
```js
             'Cannot find any files matching pattern'
```

Reply:
Good idea.

Reply:
Hmm I actually think listing the errors is better, from the users perspective it's more helpful and easier to see the mistake that way. I've used it myself during his pr.

Reply:
So if user gives 20 globs and _none_ exist, one message per is somehow more informative than just saying "none found"? I disagree. 
For warnings, fine -- but not when _none_ exist.

Reply:
As @craigtaub stated, it was useful information, so if some *subset* of the paths are not found, it seems reasonable to list them (but not if *all* of them are not found).

FWIW, a user can avoid this problem altogether by using shell expansion, but that's not portable.

Reply:
If a subset don't exist, those would be warnings -- stated I had no problem with that.
Just the "none found" error case.

Reply:
Ok i think its clear you both feel the single error message is better than multiple error messages. For debugging purposes I found the multiple one better, the initial idea of using an error was to indicate nothing did run, but perhaps its not clear enough.